docs/docs/architecture/keys.md

```diff
@@ -33,9 +33,9 @@ The remote attestation statement of a Constellation cluster combines *baseID* an
 Constellation encrypts all cluster network communication using the [container network interface (CNI)](https://github.com/containernetworking/cni).
 See [network encryption](networking.md) for more details.
 
-The Cilium agent running on each cluster node will establish a secure WireGuard tunnel between it and all other known nodes in the cluster.
-Each node automatically creates its own [Curve25519](http://cr.yp.to/ecdh.html) encryption key-pair and distributes its public key via Kubernetes.
-Each node’s public key is then used by other nodes to decrypt and encrypt traffic from and to Cilium-managed endpoints running on that node.
+The Cilium agent running on each node establishes a secure [WireGuard](https://www.wireguard.com/) tunnel between it and all other known nodes in the cluster.
+Each node creates its own [Curve25519](http://cr.yp.to/ecdh.html) encryption key pair and distributes its public key via Kubernetes.
+A Node uses another node's public key to decrypt and encrypt traffic from and to Cilium-managed endpoints running on that node.
```

Comment:
```suggestion
A node uses another node's public key to decrypt and encrypt traffic from and to Cilium-managed endpoints running on that node.
```
We have been using lowercase "node" here. Not sure if there are other parts of the documentation where we used uppercase "Node". Might make sense when we use the "Kubernetes Node" term similar to "Pod". It's a wild mix in the k8s docs as well...

Reply:
Changed to lowercase. Uppercase was actually not on purpose.